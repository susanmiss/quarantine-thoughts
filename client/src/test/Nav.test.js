import React from 'react';
import ReactDOM from 'react-dom'
import {shallow} from 'enzyme'; 
import renderer from 'react-test-renderer';
import Nav from '../Nav'


const setup = (props={}, state=null) => {
    const wrapper = shallow(<Nav {...props} />)
    if( state ) wrapper.setState(state);
    return wrapper;
  }

  const findByTestAttr = (wrapper, val) => {
    return wrapper.find(`[data-test="${val}"]`);
  }

describe("<Nav />", () => {


  let wrapper;
  beforeEach(() => wrapper = shallow(<Nav />));
  
  it('Renders correctly', () => {
      expect(wrapper).toMatchSnapshot();
  })
  

 
})