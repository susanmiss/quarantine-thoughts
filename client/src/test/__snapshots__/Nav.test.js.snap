// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Nav /> Renders correctly 1`] = `ShallowWrapper {}`;
